.github/workflows/builds.yml

```diff
@@ -111,6 +111,22 @@ jobs:
       - name: Downstream run
         run: ./build-downstream/bin/ShowActsVersion
 
+      # We configure and build without ROOT in the same job to garantuee that we can use the ccache cache
+      - name: Configure (without ROOT)
+        run: >
+          ccache -z &&
+          rm -rf build &&
+          cmake -B build -S .
+          --preset=github-ci-no-root
+          -DCMAKE_INSTALL_PREFIX="${INSTALL_DIR}"
+          -DPython_EXECUTABLE=$(which python3)
```

Comment:
Maybe we can also disable the unit tests for this config? That shouldn't make a difference for what we're trying to test here, but I think we spend the majority of the time in this step on linking the unit test executables, since that's not sped up by ccache.

Reply:
I actually found a bug because unit test where switched on (we have some examples unit test using root)

Reply:
But that should be caught in the first run, shouldn't it?